build/yamls/antrea-windows-containerd.yml

```diff
@@ -9,6 +9,8 @@ data:
     mkdir -force C:/var/run/secrets/kubernetes.io/serviceaccount
     cp $mountPath/var/run/secrets/kubernetes.io/serviceaccount/ca.crt C:/var/run/secrets/kubernetes.io/serviceaccount
     cp $mountPath/var/run/secrets/kubernetes.io/serviceaccount/token C:/var/run/secrets/kubernetes.io/serviceaccount
+    mkdir -force c:/opt/cni/bin/
```

Comment:
As the directory is created in the initContainers, the path may be deleted when Pod is deleted. Is it as expected?

Reply:
really? then how could `cp $mountPath/k/antrea/cni/* c:/opt/cni/bin/` below take effect if `c:/opt/cni/bin/` is not in host filesystem?

Reply:
Yes they are created on in host filesystem and deleting pod won't delete these directories.

Reply:
It should by my fault. Before it can be merged, I also want to know the difference between this solution and the one that uses hostPath in container volumes?

Reply:
I don't know how this actually works because there is no configuration saying `c:/` is a host volume, I guess Windows hostProcess Pod is different from Linux Pod and it has direct access to host filesystem. Then perhaps there is no point to leverage hostPath to achieve it just for these two directories given the script already creates other directories in host filesystem.